machine_test.go

```diff
@@ -333,7 +334,6 @@ func TestMicroVMExecution(t *testing.T) {
 	case <-exitchannel:
 		// if we've already exited, there's no use waiting for the timer
 	}
-	t.Run("TestStopVMM", func(t *testing.T) { testStopVMM(ctx, t, m) })
```

Comment:
What's the rationale for removing this test?

Reply:
isn't removing it, moving it above that way it occurs before exitchannel. 

Also fixes a bug where `StopVMM` was racing in `TestMicroVMExecution` and would sometimes succeed because the defer would happen last and do nothing with the error. However, if the defer happened before the `TestStopVMM`, then that's when the error is captured and fails.

Reply:
The point of having `TestStopVMM` in the current location is so that the VM does start up completely before we try to stop it; see line 330-336.  Do you believe that we no longer need to test this scenario?

Reply:
The other option is having the test run in a `defer` and removing `defer StopVMM()` as well. That would keep to ensure that testing shutdown of a fulling started VM is still there. Thoughts?